### PR TITLE
Added maxRequestSize to ServerConfig to specify size of incoming req

### PR DESCRIPTION
**Pull request by claesloof**
It is currently possible to specify maxResponseSize in HttpClientConfig for responses. This PR lets you specify a maxRequestSize for _incoming_ requests.

### CODE REVIEW

File: jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestHandler.java [hunk 1]
```diff
@@ -18,30 +18,34 @@
...
         this(services.getResources(),
             jaxRsResourceFactory,
             exceptionHandler,
+            collector,
             null);
     }
 
     public JaxRsRequestHandler(Object... services) {
-        this(services, new JaxRsResourceFactory(), new ExceptionHandler(), null);
+        this(services, new JaxRsResourceFactory(), new ExceptionHandler(), new ByteBufCollector(10*1024*1024), null);
```
Comment (JonasHallFnx) [inline jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestHandler.java]: Perhaps make a default constructor in ByteBufCollector that makes it 10Mb, so you can remove all thos 10*1024*1024 spread in the code?

File: jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestHandler.java [hunk 2]
```diff
@@ -18,30 +18,34 @@
 @Singleton
 public class JaxRsRequestHandler implements RequestHandler<ByteBuf, ByteBuf> {
 
-    private JaxRsResources resources;
-
+    private JaxRsResources   resources;
     private ExceptionHandler exceptionHandler;
+    private ByteBufCollector collector;
```
Comment (JonasHallFnx) [inline jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestHandler.java]: final?

File: jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestHandler.java (hunk 1 above)
Comment (claesloof) [reply jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestHandler.java]: Yeah, that would be much cleaner.

File: jaxrs/src/test/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestTest.java [hunk 3]
```diff
@@ -21,35 +22,68 @@ public void shouldDecodeMultiChunkBody() {
...
+        JaxRsRequest          req       = new JaxRsRequest(serverReq, null, new byte[0], new ByteBufCollector(10*1024*1024));
         assertThat(req.getPathParam("path")).isNull();
         assertThat(req.getQueryParam("query")).isNull();
         assertThat(req.getFormParam("form")).isNull();
         assertThat(req.getHeader("header")).isNull();
         assertThat(req.getCookie("cookie")).isNull();
     }
+
+
+    private String generateLargeString(int sizeInMB) {
```
Comment (jepp3) [inline jaxrs/src/test/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestTest.java]: maybe do something like this?

```java
private String generateLargeString(int sizeInMB) {
		String largeString = IntStream.range(1, sizeInMB * 1024 * 1024)
			.mapToObj(i -> "b")
			.collect(Collectors.joining());

		return "\"" + largeString + "\"";
	}
```

File: jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestHandler.java (hunk 2 above)
Comment (claesloof) [reply jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestHandler.java]: Suppose they all could be final then?

File: jaxrs/src/test/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestTest.java (hunk 3 above)
Comment (claesloof) [reply jaxrs/src/test/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestTest.java]: Nice! Worked like a charm.

**Pull request merged**